packages/plugin-ext/src/main/browser/text-editor-service.ts

```diff
@@ -43,6 +43,7 @@ export class TextEditorServiceImpl implements TextEditorService {
     constructor(@inject(EditorManager) private editorManager: EditorManager) {
         editorManager.onCurrentEditorChanged(this.onEditorChanged);
         editorManager.onCreated(w => this.onEditorCreated(w));
+        editorManager.all.forEach(w => this.onEditorCreated(w));
```

Comment:
@tolusha thanks for tackling this! 🙏 

in general this should work, but we could actually clean up the `editorManager.onCurrentEditorChanged(this.onEditorChanged)` here, no?

WDYT of renaming `onEditorCreated` to `connectEditor` or something like that, adjusting `onCurrentEditorChanged` handler, and now calling `connectEditor` for `editorManager.currentEditor` once?

Reply:
maybe a bit off topic as this is not related to the issue in scope of this PR, but before I forget to mention it: I think we should remove `this.editors` map which holds references to `MonacoEditor` instances without a real reason.

e.g. `getActiveEditor` uses `editorManager` an so could `listTextEditors` be implemented as well.

Reply:
@AlexTugarev 
I've done some refactoring as you mentioned. 
I am not sure if changes regarding `connectEditor` make sense.

Reply:
ok with me. let's test with many opened editors to see how it works with many events being fired.